CCAMS.vcxproj

```diff
@@ -35,11 +35,12 @@
     <PlatformToolset>v143</PlatformToolset>
     <PreferredToolArchitecture>x86</PreferredToolArchitecture>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release-Static|Win32'" Label="Configuration">
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release-static|Win32'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <CharacterSet>MultiByte</CharacterSet>
     <PlatformToolset>v143</PlatformToolset>
     <PreferredToolArchitecture>x86</PreferredToolArchitecture>
+    <UseOfMfc>false</UseOfMfc>
```

Comment:
This should be true if I am not wrong - ES SDK relies on MFC

Reply:
Indeed, it should be fixed now, I cleaned up `AdditionalDependencies` to remove WinInet (not needed if we use `httplib`) but had to list a few others (namely `User32.lib`, `Advapi32.lib` and `Crypt32.lib`)